pkg/investigations/clustermonitoringerrorbudgetburn/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+name: clustermonitoringerrorbudgetburn
+rbac:
+  roles: []
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "config.openshift.io"
+      resources:
+        - clusteroperators
+customerDataAccess: true
```

Comment:
Question: does the true here have any effect? Also, would false not be more accurate here?

Reply:
its used only for lockbox clusters to decide if we can go ahead

Reply:
yeah false sounds better here, updating..